source/features/bugs-tab.tsx

```diff
@@ -24,14 +24,46 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
+const bugLabelCacheKey = (): string => 'bugs-label:' + getRepo()!.nameWithOwner;
+const bugLabel = async (): Promise<string | undefined> => cache.get<string>(bugLabelCacheKey());
+
 const countBugs = cache.function(async (): Promise<number> => {
-	const {search} = await api.v4(`
-		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
-			issueCount
+	if (!await bugLabel()) {
+		const {repository} = await api.v4(`
+			repository() {
+				labels(query: "bug", first: 10) {
+					nodes {
+						name
+						issues(states: OPEN) {
+							totalCount
+						}
+					}
+				}
+			}
+		`);
+
+		const {nodes}: AnyObject = repository.labels;
+		if (nodes.length === 0) {
+			return 0;
+		}
+
+		const bugTypes = new Set([':bug: bug', 'bug', 'confirmed-bug', 'type: bug']);
+		const {name, issues} = nodes.find((label: AnyObject) => bugTypes.has(label.name.toLowerCase())) ?? {};
```

Comment:
```suggestion
		const bugTypes = new Set([':bug:bug', 'bug', 'confirmed-bug', 'type:bug']);
		const {name, issues} = nodes.find((label: AnyObject) => bugTypes.has(label.name.toLowerCase().replace(/\s/g, ''))) ?? {};	
```

Or change it to a regex: `/^(:bug:bug|bug|confirmed-bug|type:bug)$/` or something long those lines, we don't need to make it too complex

Reply:
> we don't need to make it too complex

Maybe we do a regex to include `:`word`:`(optinalSpace)`bug` like this we can include all emoji  bug labels?

Reply:
The `.replace(/\s/g, '')` part already does away with all the spaces, that's why my suggestions don't have it.

As for the `:\w+:` emoji, sure